packages/node_modules/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -127,9 +127,14 @@ const Authorization = WebexPlugin.extend({
    * @returns {Promise}
    */
   initiateLogin(options = {}) {
+    if (options.email) {
+      options.emailHash = CryptoJS.SHA256(options.email).toString();
+    }
+    delete options.email;
```

Comment:
You should not modify arguments of the function. You may create a copy of the arguments and modify that but do not introduce side-effects on this function.

Reply:
Is it specifically the `delete` you are objecting to for this function, or the way the function works overall? Whilst I agree that in general function arguments should not be modified (unless actually intended), this particular function has always modified `options`, for example adding `state` (and attributes of it) if it didn't exist previously, or more recently adding in the `code_challenge`.

Reply:
Overall; I know this was modified recently by Colin and we didn't catch that.
I don't agree that we should keep doing something that we know it's not ok just because that's how it was done in the past. Let's do better with our code changes.
Creating an object copy and modifying that would satisfy my request